fivox/eventSource.cpp

```diff
@@ -166,18 +163,10 @@ void EventSource::add( const Event& event )
     _impl->events.push_back( event );
 }
 
-void EventSource::update( const size_t index, float value )
+void EventSource::update( const size_t index, const float value )
```

Comment:
Right, the update changes only the value. The radius is not modified, it's only set at construction time (same as the position). I'm not sure I get what you suggest... add setters for all three members (position, radius, value)? or add more parameters to the update function?